debian/patches/use_system_libyaml

```diff
@@ -0,0 +1,58 @@
+Description: Use system libyaml
+Author: Faidon Liambotis <paravoid@debian.org>
```

Comment:
Separately from that: Though I don't work for twitter, this repo still requires the CLA for contributions, and these files are authored by someone else - https://cla-assistant.io/twitter/twemproxy

Though how that could be fixed is irrelevant since I don't plan on including the changes made for debian packagers in this repo itself for the previously stated reasons.

> 5. You represent that each of Your Contributions is **Your original creation** (see section 7 for submissions on behalf of others). You represent that Your Contribution submissions include complete details of any third-party license or other restriction (including, but not limited to, related patents and trademarks) of which you are personally aware and which are associated with any part of Your Contributions.

> 7. Should You wish to submit work that is not Your original creation, You may submit it to Twitter separately from any Contribution, identifying the complete details of its source and of any license or other restriction (including, but not limited to, related patents, trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work as "Not a Contribution. Third-party materials licensed pursuant to: [license name(s) here]" (substituting the bracketed text with the appropriate license name(s)).